src/routes/transactions/entities/transaction-details/transaction-details.entity.ts

```diff
@@ -46,4 +46,6 @@ export class TransactionDetails {
   txHash!: string | null;
   @ApiPropertyOptional({ type: SafeAppInfo, nullable: true })
   safeAppInfo!: SafeAppInfo | null;
+  @ApiPropertyOptional({ type: String, nullable: true })
+  note?: string;
```

Comment:
The question mark indicates that the field can be undefined if the value is not provided. However, to ensure the value is explicitly null and for consistency, we should define it as: `note!: string | null;`

Reply:
I've made the change but IMHO this convention is a cargo cult that started with the original Rust CGW and the Python-based transaction-service not having an undefined type that we blindly carried it over to this JavaScript project. There's very little value in sending these empty payloads to the client and having to write non-idiomatic null-checking code for them.

Reply:
@katspaugh Thanks for updating the code! I agree that we can revisit it later, but let’s create a task and address it in a separate PR.